Makefile

```diff
@@ -38,12 +38,16 @@ ifneq ($(TAGS),)
 BUILDTAGS = -tags '$(TAGS)'
 endif
 
-build:
+build: tooldep
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
-install:
+install: tooldep
 	cd $(INSTALL_DIR) && $(GO) install $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS)
 
+tooldep:
+	@echo "see if mtools is installed on the system"
```

Comment:
I'm disabling the standard echoing of the commands, however I have a feeling you're gonna shoot this  down I think.